composer.json

```diff
@@ -21,15 +21,16 @@
         "source": "https://github.com/xiidea/EasyAuditBundle"
     },
     "require": {
-        "php": "^7.2.5",
+        "php": "^8.0.0",
         "psr/log": "^1",
         "symfony/framework-bundle": ">=5.0 <6.0.0",
-        "symfony/security-bundle": ">=5.0 <6.0.0"
+        "symfony/security-bundle": ">=5.0 <6.0.0",
+        "symfony/symfony": "5.2.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
+        "phpunit/phpunit": "^9.5.2",
         "php-coveralls/php-coveralls": "^2.1",
-        "doctrine/common": ">=2.2 <3.0"
+        "doctrine/common": "^3.1.1"
```

Comment:
Dependency should be defined in range, like doctrine-bundle, otherwise we may not know for any changes in doctrine/common@3.2.0 causing the lib not to work.

Reply:
removed symfony dependency